src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -521,13 +523,15 @@ protected void addTracerTag(String key, Object value) {
     /**
      * Internal logging.
      */
+    @SuppressWarnings("WeakerAccess")
     protected void debug(String s) {
         debug(s, null);
     }
 
     /**
      * Internal logging.
      */
+    @SuppressWarnings({"WeakerAccess", "SameParameterValue"})
```

Comment:
This is just for my own understanding/curiosity - do you happen to know why there is a `SameParameterValue` warning? The parameters being passed in don't appear to be constants...